### DISCUSSION TITLE
Afficher un planning hebdomadaire dans une fenêtre synapps

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#571] [General]
Bonjour, 
J'aimerais afficher un planning hebdomadaire dans Synapps pour que le client puisse modifier simplement.
![image](https://user-images.githubusercontent.com/46195964/220938414-736557d5-d45c-49fa-91c8-b8132ce8fee7.png)

Ici : 

![image](https://user-images.githubusercontent.com/46195964/220938911-0c8b25d7-f7b6-41cb-9ec2-8966617e76b6.png)

**Comment by WITNicolas** [comment]
Bonjour, petit complément
La question porte sur le planning hebdo et non pas l'agenda.

Voici le fil de discussion dans lequel vous trouverez le composite en question https://github.com/witsa/synapps/discussions/471
Il suffit de copier et de coller dans le menu composite (bouton droite puis coller)

**Reply by FlorentAdell** [reply]
Bonjour, merci Nicolas pour la réponse. 

Florent,
Solubat
